raiden_contracts/tests/fixtures/token.py

```diff
@@ -39,6 +39,6 @@ def f(initial_amount: int, decimals: int, token_name: str, token_symbol: str):
 
 
 @pytest.fixture
-def standard_token_contract(deploy_token_contract):
+def standard_token_contract(custom_token):
     """Deployed HumanStandardToken contract"""
```

Comment:
Then maybe fix the description to say `"""Deployed CustomToken contract"""`, so it's not confusing.